<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvokeClass</class>
 <widget class="QMainWindow" name="InvokeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoke</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 91, 126);
background-color: rgb(35, 35, 35);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="connectRemote_GroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Run Remote Apps</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>298</width>
       <height>93</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="gateway_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Gateway Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="P3_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Prepar3D Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Stage_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>Stage Path</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="gateway_lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="runGateway">
            <property name="styleSheet">
             <string notr="true">color: rgb(49, 49, 49);
background-color: rgb(255, 183, 135);</string>
            </property>
            <property name="text">
             <string>run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="P3_lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="runP3">
              <property name="styleSheet">
               <string notr="true">color: rgb(49, 49, 49);
background-color: rgb(255, 183, 135);</string>
              </property>
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="stage_lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="runStage">
              <property name="styleSheet">
               <string notr="true">color: rgb(49, 49, 49);
background-color: rgb(255, 183, 135);</string>
              </property>
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 65, 65);
font: 8pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Server State:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="serverConnect">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Nirmala UI Semilight&quot;;
</string>
        </property>
        <property name="text">
         <string>Offline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startServer">
        <property name="styleSheet">
         <string notr="true">color: rgb(49, 49, 49);
background-color: rgb(255, 183, 135);</string>
        </property>
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="invoke.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_runGatewayClicked()</slot>
  <slot>on_runStageClicked()</slot>
  <slot>on_runP3Clicked()</slot>
  <slot>on_getRemoteIPClicked()</slot>
 </slots>
</ui>
